### PR TITLE
Update Assembly File Version

### LINKED ISSUE #2244: Update AssemblyFileVersion
The `AssemblyFileVersion` attribute should now be updated to read `4.1.0.0` for all `AssemblyInfo.cs` files within this repository for the 4.1 milestone.

### PR DESCRIPTION
**Pull request by BHoMBot**
Closes #2244

### CODE REVIEW

**Review by FraserGreenroyd — APPROVED** [review]
Reviewed from a CI/CD perspective.

File: .ci/code/Serialiser_Test/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -32,5 +32,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("4.0.0.0")]
+[assembly: AssemblyFileVersion("4.1.0.0")]
```
Comment (IsakNaslundBh) [inline .ci/code/Serialiser_Test/Properties/AssemblyInfo.cs:36]: Assume this is correct? As in, we keep the ci test ones on the same version as the rest?

**Review by IsakNaslundBh — APPROVED** [review]
LGTM!

**Pull request merged**